TowerDefense/GameObjects/Node.cs

```diff
@@ -40,12 +40,26 @@ public Node(Point actualPos, Point simplePos, Texture2D tex) : base(tex, actualP
         {
             this.actualPos = actualPos;
             this.simplePos = simplePos;
+
+            spriteHeight = 32;
+            spriteWidth = 32;
+            currentFrame = 0;
+        }
+
+        public override bool Update(GameTime gameTime)
+        {
+            frameCount = portal ? 4 : 0;
+            currentFrame = portal ? currentFrame : 0;
```

Comment:
I think it's clear I just need portal / wall / cheese classes, since specifics of them are sprinkled all over the Node class, such as the fact that portals have 4 animation frames, and the cost of them is hard-coded throughout.